src/stories/2-components/01-Layout.stories.js

```diff
@@ -36,7 +36,21 @@ containerNarrow.story = {
   parameters: {
     info: {
       text:
-        'A narrow element that aligns to the left side of the page and is useful to wrap content.',
+        'A narrow element that aligns to the left side of the page and is useful to wrap long content within a full container.',
+    },
+  },
+}
+
+export const containerCentered = () => (
+  <Container centered>
+    <SampleContainer />
+  </Container>
+)
+containerCentered.story = {
+  parameters: {
+    info: {
+      text:
+        'A contered container, the default container for long-form content..',
```

Comment:
* See Typo CI
* Can you note that the centered container also uses the narrow styles?

Reply:
Typo CI finally caught something! I'll fix this.